client/state/selectors/is-jetpack-onboarding-step-completed.js

```diff
@@ -0,0 +1,48 @@
+/** @format */
+
+/**
+ * External dependencies
+ */
+import { get } from 'lodash';
+import { translate } from 'i18n-calypso';
+
+/**
+ * Internal dependencies
+ */
+import createSelector from 'lib/create-selector';
+import { getJetpackOnboardingSettings } from 'state/selectors';
+import { JETPACK_ONBOARDING_STEPS as STEPS } from 'jetpack-onboarding/constants';
+
+export const isJetpackOnboardingStepCompleted = createSelector(
```

Comment:
Why the named export? `default` should be sufficient, as per our one-selector-per-file-in-`state/selectors` convention, no?

Reply:
Good point, fixed in https://github.com/Automattic/wp-calypso/pull/21438/commits/55b5451ccd1a4f1141b24c26cec0da9893f55e35.